pyneuroml/plot/PlotSpikes.py

```diff
@@ -36,9 +36,12 @@
 POP_NAME_SPIKEFILE_WITH_GIDS = "Spiketimes for GIDs"
 
 
-def process_args():
+def process_args() -> argparse.Namespace:
```

Comment:
I think we should make this `_process_args` to indicate that it's an internal method and not part of the public API?